pkg/resource/replication_group/testdata/test_suite.yaml

```diff
@@ -0,0 +1,23 @@
+tests:
+  -
+    name: "rg_snapshot_cmd"
+    description: "ReplicationGroup snapshot cluster mode disabled tests"
+    scenarios:
+      -
+        name: "ReadOne=NoDiff"
+        description: "Given desired state matches with server side resource data, ko.Status remain unchanged."
```

Comment:
This test description doesn't mention anything about a snapshot, which is odd considering `snapshot` is part of the test name.

Reply:
Updated the test file names, and resource name to indicate cluster mode disabled.